src/azure-cli/azure/cli/command_modules/servicefabric/commands.py

```diff
@@ -169,10 +163,10 @@ def load_command_table(self, _):
         g.custom_command('add', 'add_client_cert')
         g.custom_command('delete', 'delete_client_cert')
 
-    with self.command_group('sf managed-node-type', node_type_mgmt, is_preview=True,
+    with self.command_group('sf managed-node-type', node_type_mgmt, is_preview=False,
```

Comment:
You can also delete `is_preview=False`. Default is `False`.

Reply:
remvoed thanks